example-client/README.md

```diff
@@ -3,11 +3,219 @@ Example CoAP client
 
 This is an example coap client application, that can be use with any LwM2M Server.
 
+# Running
+
 Usage
 -----
 
     ./run.sh <registration-url> [<keystore-file>]
 
+note: parameters are parsed like a [www URL query string](https://en.wikipedia.org/wiki/Query_string) where ? and & are delimeters for the endpoint name and account ID
+
 *For example:*
-        
-    ./run.sh 'coap://localhost:5683/rd?ep=device007&lt=3600&b=U'    
+    
+    cd java-coap/example-client
+
+    ./run.sh "coaps://lwm2m.us-east-1.mbedcloud.com:5684/rd?ep=${ENDPOINT_NAME}&aid=${ACCOUNT_ID}" ../../keystore/example-client-device-keystore.jks
```

Comment:
if it's example, then provide example for ENDPOINT_NAME and ACCOUNT_ID, in other section, describe that for mbed cloud that the registration pattern

Reply:
${ENDPOINT_NAME} and ${ACCOUNT_ID} is defined in a variable in the Configuration section below.  Users can do things like 'export ENDPOINT_NAME=myEPname' or source .env.

Reply:
This example is at the beginning of the document but Configuration section way below.. Wouldn't it make more sense to either:
- place this example after Configuration section,
- or provide more complete example

Reply:
I wrestled with that too.  I concluded that it might make it easier for someone who new what they were doing to just have the runs steps up front and the config steps needed for mbed cloud following.  Whatever you think makes the most sense is fine with me.